website/src/blog/2024-08-30-hot-chocolate-14/2024-08-30-hot-chocolate-14.md

```diff
@@ -396,13 +396,13 @@ The optional argument on the `ParentAttribute` specifies a selection set that de
 
 Apart from this, we have invested a lot into `GreenDonut` to ensure that you can use the source-generated `DataLoader` without any dependencies on `HotChocolate`. Since `DataLoader` is ideally used between the business layer and the data layer and is transparent to the REST or GraphQL layer.
 
-With Hot Chocolate 14, you can now add the `HotChocolate.Types.Analyzers` package and the `GreenDonut` package to your data layer. The analyzers package is just the source generator and will not be a dependency of your package. We will generate the `DataLoader` code plus the dependency injection code for registering your `DataLoader`. You simply need to add the `DataLoaderModuleAttribute` to your project like the following:
+With Hot Chocolate 14, you can now add the `HotChocolate.Types.Analyzers` package and the `GreenDonut` package to your data layer. The analyzers package is just the source generator and will not be a dependency of your own package. We will generate the `DataLoader` code plus the dependency injection code for registering your `DataLoader`. You simply need to add the `DataLoaderModuleAttribute` to your project like the following:
 
 ```csharp
 [assembly: DataLoaderModule("CatalogDataLoader")]
 ```
 
-Lastly, on the topic of DataLoader we have made the DataLoader cache observable allowing you to share entities between DataLoader for even more efficient caching. Lets for say that we have two brand DataLoader, one fetches the entity by id and the other one by name.
+Lastly, on the topic of DataLoader we have made the DataLoader cache observable allowing you to share entities between DataLoader for even more efficient caching. Lets for instance say that we have two brand DataLoader, one fetches the entity by id and the other one by name. How can we make sure that we do not fetch the same entity twice just because we have different keys?
```

Comment:
```suggestion
Lastly, on the topic of DataLoader, we have made the DataLoader cache observable, allowing you to share entities between DataLoader for even more efficient caching. Let's for instance say that we have two Brand DataLoader, one fetches the entity by ID and the other one by name. How can we make sure that we do not fetch the same entity twice just because we have different keys?
```

I'm also 99% sure that `two Brand DataLoader` should be `two Brand DataLoaders`.